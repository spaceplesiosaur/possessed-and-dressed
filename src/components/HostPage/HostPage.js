import React from 'react'
import './HostPage.scss'

export const HostPage = ({host}) => {
  return (
    <section className="hostPage-main-frame">
      <p className="host-name-tag">Meet {host.name}! {host.name} is very excited to be possessed by you.</p>
      <img className="hostPage-picture-host" src={host.picture} alt="hostPage-main-hostPic"></img>
    </section>
  )
}

export default HostPage;
