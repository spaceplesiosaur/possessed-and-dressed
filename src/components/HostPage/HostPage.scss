$dark-text-color: #282c34;
$bright-header-color: #FA1E88;
$dark-subHeader-color: #541A8B;

.hostPage-picture-host {
  border-radius: 5px;
  border: 1px $dark-text-color solid;
  margin: 2rem;
  text-align: center;
  box-shadow: 0 4.5px 9px 0 rgba(250, 30, 136, 0.25), 0 7px 25px 0 rgba(0, 0, 0, 0.2);
  max-height: 640px;
  max-width: 400px;
}

.host-name-tag {
  color: $dark-text-color;
  font-size: 4rem;
  margin: 4rem;
}
