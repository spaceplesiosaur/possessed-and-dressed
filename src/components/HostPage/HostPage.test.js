import React from 'react';
import HostPage from './HostPage';
import { shallow } from 'enzyme';

describe('HostPage', () => {
  it('should render with correct data', () => {
    const mockHost = {
        id: 1,
        name: "Blucifer",
        picture: "https://i.imgur.com/I5MqMUV.jpg",
        season: 9
    }
    const wrapper = shallow(<HostPage host={mockHost}/>)

    expect(wrapper).toMatchSnapshot()
  })
})
