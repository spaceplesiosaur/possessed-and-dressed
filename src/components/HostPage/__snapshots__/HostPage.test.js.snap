// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HostPage should render with correct data 1`] = `
<section
  className="hostPage-main-frame"
>
  <p
    className="host-name-tag"
  >
    Meet 
    Blucifer
    ! 
    Blucifer
     is very excited to be possessed by you.
  </p>
  <img
    alt="hostPage-main-hostPic"
    className="hostPage-picture-host"
    src="https://i.imgur.com/I5MqMUV.jpg"
  />
</section>
`;
